src/render.js

```diff
@@ -19,7 +19,7 @@ function resolveComponents(vnode) {
     }
 }
 
-export function renderToDiagram(vnode, co = false) {
+export function renderToDiagram(vnode, align = false, co = false) {
```

Comment:
Please use an `options` object as discussed in https://github.com/yishn/tikzcd-editor/issues/27#issuecomment-425070268.